chanzuckerberg/czgenepi
# Maya/187986 remove old homepage

### Summary
- **What:** Remove the old homepage component and related tests/assets
- **Why:** rebrand clean up
- **Ticket:** [[sc-187986]](https://app.shortcut.com/genepi/story/187986)
- **Env:** https://rebrand-frontend.dev.czgenepi.org/

### Testing
1. Make sure the homepage still loads
2. Ensure /landingv2 returns a 404
3. Tests pass

### Demos
No visual changes.

### Checklist
